reactor-core/src/main/java/reactor/core/publisher/Hooks.java

```diff
@@ -464,6 +466,27 @@ public static void resetOnNextError() {
 		}
 	}
 
+	/**
+	 * Globally enables the {@link reactor.util.context.Context} loss detection that may happen
```

Comment:
`may` suggests that there is more conditions for it to happen. did you mean `will`?
I'd rephrase along the lines of:
> Globally enables the Context loss detection in operators like transform or transformDeferred when non-Reactor types are used.